add-ons/helm/README.md

```diff
@@ -1,41 +1,69 @@
 # Helm Add-on
-Installs [kubernetes helm](https://github.com/kubernetes/helm) tiller into minishift. Tiller is the server side part of helm.
 
-Verify you have installed these addons, by following the [general readme](../../README.adoc#download-and-use-community-add-ons).
+Don't forget to read the [general readme](../../README.adoc#download-and-use-community-add-ons).
 
-## Deploy helm tiller
-To deploy tiller on OpenShift, apply this addon:
+Tiller is the server side part of helm. You can check how to installs kubernetes helm in this [link](https://github.com/kubernetes/helm).
+
+**NOTE** Please check in the [version](version) file to confirm the current release is compatible with the helm client and tiller server.
+
+Before install helm into minishift please check if your minishift was started with defaults installed and admin-user enabled:
 
 ```
-$ minishift addon apply helm
+$ minishift addons install --defaults
+Default add-ons 'anyuid, admin-user, xpaas, registry-route' installed
+
+$ minishift addons enable admin-user
+Add-on 'admin-user' enabled
 ```
 
+After configured your minishift you can start using, for example:
+
+```
+$ minishift start --profile minishift --disk-size 40gb --memory 4096mb
+```
+
+The options above are not mandatory but show how you can define memory and disk for your local deployments.
+
+
+## Deploy and install helm tiller
+To deploy tiller on Minishift or OpenShift, apply this addon:
+
+```
+$ minishift addons install helm
+$ minishift addons apply helm
+```
+
+
 ## Discover tiller host
-Get Tiller host URL by running these commands in the shell:
+Get `oc`, `helm_host`(tiller) and `minishift` by adding this new variable into ~/.bashrc
 
 ```
-$ export TILLER_HOST="${minishift ip}:$(oc get svc/tiller -o jsonpath='{.spec.ports[0].nodePort}' -n kube-system --as=system:admin)"
+export PATH="~/.minishift/cache/oc/v3.7.1/linux:$PATH"
```

Comment:
I would suggest replace the above with `eval $(minishift oc-env)`. The above has a hard coded version and that's something we should avoid.